packages/messaging/src/controllers/sw-controller.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2017 Google Inc.
+ * Copyright 2019 Google Inc.
```

Comment:
go/copyright

It seems like this shouldn't be replaced, and you can add the current year so long as you preserve the original year? So 2017 or 2017-2019 but not 2019?

Reply:
It's a bot that adds these, I think I removed the original and it added a new one with the current (well, it was current then) year. :man_shrugging:

I'll fix them.